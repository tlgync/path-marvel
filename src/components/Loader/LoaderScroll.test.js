import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { LoaderScroll } from '.';

test('LoaderScroll Render SnapShot', () => {
  const tree = renderer.create((
    <Router>
      <LoaderScroll />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
