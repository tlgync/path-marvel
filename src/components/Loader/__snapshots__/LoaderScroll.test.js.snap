// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Render SnapShot 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "#eee",
      "bottom": 0,
      "display": "flex",
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "zIndex": 99,
    }
  }
>
  <div
    aria-label="animation"
    onClick={[Function]}
    role="button"
    style={
      Object {
        "height": "55px",
        "margin": "0 auto",
        "outline": "none",
        "overflow": "hidden",
        "width": "55px",
      }
    }
    tabIndex="0"
    title=""
  />
</div>
`;

exports[`LoaderScroll Render SnapShot 1`] = `
<div
  aria-label="animation"
  onClick={[Function]}
  role="button"
  style={
    Object {
      "height": "55px",
      "margin": "0 auto",
      "outline": "none",
      "overflow": "hidden",
      "width": "55px",
    }
  }
  tabIndex="0"
  title=""
/>
`;
